thread.c

```diff
@@ -5616,6 +5664,7 @@ rb_check_deadlock(rb_ractor_t *r)
     int sleeper_num = rb_ractor_sleeper_thread_num(r);
     int ltnum = rb_ractor_living_thread_num(r);
 
+    if (GET_THREAD()->vm->ignore_deadlock) return;
```

Comment:
Toooo trivial comment:
You can put it just after `rb_check_deadlock()` because it is C99.